Gemfile

```diff
@@ -1,4 +1,8 @@
 source 'https://rubygems.org'
 
-gem 'jekyll'
+gem 'jekyll', '~> 2.5.3'
 gem 'jekyll-redirect-from'
+
+group :jekyll_plugins do
+  gem 'algoliasearch-jekyll', '~> 0.4.3'
```

Comment:
Our plugin is not in the list of official plugins endorsed by GitHub. You have to do a `jekyll algolia push` locally whenever you do a `git push` to `gh-pages` to keep data in your search index in sync with data really displayed in your site.